tools/fast/tests/slashing_test.go

```diff
@@ -0,0 +1,108 @@
+package tests
+
+import (
+	"context"
+	"fmt"
+	"math/big"
+	"testing"
+	"time"
+
+	files "github.com/ipfs/go-ipfs-files"
+	"github.com/pkg/errors"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+
+	"github.com/filecoin-project/go-filecoin/actor/builtin/miner"
+	"github.com/filecoin-project/go-filecoin/address"
+	"github.com/filecoin-project/go-filecoin/protocol/storage/storagedeal"
+	tf "github.com/filecoin-project/go-filecoin/testhelpers/testflags"
+	"github.com/filecoin-project/go-filecoin/tools/fast"
+	"github.com/filecoin-project/go-filecoin/tools/fast/fastesting"
+	"github.com/filecoin-project/go-filecoin/tools/fast/series"
+	"github.com/filecoin-project/go-filecoin/types"
+)
+
+// Too slow to let run in CI. Use for debugging locally.
```

Comment:
It would be worth looking for an issue in the backlog or filing a new one that tracks work for parameter setting.  Running the actual code with a much lower proving period would be very useful for CI.  We already do something like this for blocktime, generalizing seems worth thinking about anyway.

Reply:
I should have taken this comment out since it's running in CI now, but I think the best issue for that would be #3180

Reply:
and it was, my "that" referred to tracking work for parameter setting specifically for proving period.